surveys/2021-contributor-survey/questions.md

```diff
@@ -0,0 +1,495 @@
+> areas / achieve new goals.
+>
+> This was one of the best received questions during the pre-survey interviews
+> and seemed to elicited thoughtful answers.
+
+### My experience contributing to the Rust Project so far has been positive.
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Useful for sentiment analysis and gagues the severity of the problems identified
+> in the subsequent questions.
+
+### I feel positive about the project leadership and governance structure.
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Suggested by @manishearth. Useful for identifying problems within project governance.
+
+### I feel positive about my own leadership within the project.
+
+**Skip if not a leader of a team or group**
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Suggested by @manishearth. Useful for identifying problems within project governance.
+
+### I feel my team/working group/project group is healthy.
+
+**Skip if not a member of the Rust Project.**
+
+- strongly agree
+- agree
+- neither agree nor disagree
+- disagree
+- strongly disagree
+
+> **justification**
+>
+> Useful for sentiment analysis and gagues the severity of the problems identified
+> in the subsequent questions.
+
+### Are you burnt out or do you feel you're at risk of burning out?
+
+- I am burnt out.
+- I am at risk of burning out.
+- I am not burnt out but I have been in the past.
+- I have not been burnt out.
+
+> **justification**
+>
+> Answers a specific question we have.
+>
+> * Is burnout a major issue across the Rust Project?
+
+### What are the biggest issues you'd like to see fixed that are negatively impacting your ability to contribute to and/or maintain Rust?
+
+Select all that apply:
+
+- There aren't enough experts or they aren't available to guide me through my contribution.
+- I don't have the financial resources to dedicate spare time to the project.
+- It takes too long to compile or test the project's code.
+- I don't have sufficiently powerful hardware to build or test the project's code.
+- The contribution process is difficult to navigate or poorly documented.
+- It takes too long for my PRs to get reviewed.
+- There are too many PRs to review for me to keep up with.
+- I don't have enough time to dedicate to the project to complete the work I wish to do.
+- I am having trouble recruiting new team members and reviewers for the portion of the project I maintain.
+- Other: Free Form
+
+> **justification**
+>
+> Core survey question. This directly identifies the problems the Rust
+> Foundation and Project leadership should be focused on.
+
+### What kind of support would you like to see or expect to see from the Rust Foundation?
+
+Select all that apply:
+
+Free Form
```

Comment:
Do we have ideas that we want to ask about (and provide a free form 'other' option) or is free form really the best way to collect answers?

Reply:
My instinct was that for this question free form would be best since the goal here is largely to gather new ideas, I worry that providing options here would significantly reduce the variety of responses we get and would also set expectations which upset people if we don't follow up on them.